fake_object_generator:
  frame_id: "/world"
  object_flag: 1
  person_flag: 1
  object_list: [[-28.61, -16.46, 0.1, -29.75, -22.58, 0.1, -35.78, -20.72, 0.1, -33.92, -14.96, 0.1],[-31.41, -35.40, 0.1, -33.43, -43.28, 0.1, -28.49, -45.18, 0.1, -26.39, -36.07, 0.1]]
  person_list: [[-24.99, -15.88, 0.1, -25.39, -16.91, 0.1, -24.04, -17.22, 0.0, -23.67, -15.76, 0.1]]
  